sphinx_source/Ucuncu_taraf_moduller/django_modülü/django_2

```diff
@@ -0,0 +1,484 @@
+.. meta::
+   :description: Bu bölümde django API'ını öğreneceğiz.
+   :keywords: python, django, çeviri
+   
+.. highlight:: python3
+
+*********************************
+İLK DJANGO PROJENİ YAZ, part 2
+*********************************
+
+Veritabanı Kurulumu
+********************
+
+Şimdi **mysite/settings.py** dosyasını açın. Önce TIME_ZONE 
+değerini `Europe/Istanbul`  yaparak İstanbul'a ayarlayın.
+
+..Note:: Dosyanın en başındaki INSTALLED_APPS ayarına dikkat edin.
+         Projenizde kullandığınız appleri tutar. Yeni bir app 
```

Comment:
'app' yerine 'uygulama' kullanılmalı. Bu metnin geri kalanındaki kullanımlar için de geçerli.